elao.app.docker/Makefile.dist

```diff
@@ -35,16 +36,67 @@ install@staging:
 	# # Composer
 	# composer install --ansi --verbose --no-interaction --no-progress --prefer-dist --optimize-autoloader --no-scripts
 	# # Npm
-	# npm install --color=always --no-progress --no-audit
+	# npm install --color=always --no-progress --no-audit --no-fund
 	# # Yarn
 	# yarn install --color=always --no-progress
 
+############
+# Database #
+############
+
+## Database - Init (create+schema)
+db.install: db.create db.update-force
+
+## Database - Create (db only)
+db.create:
+	symfony console doctrine:database:create --if-not-exists
+
+## Database - Migrate (run migrations)
+db.migrate:
+	# If you use Messenger, without auto-setup nor a migration:
+	# Messenger:
+	# symfony console messenger:setup-transports
+	# Doctrine:
+	symfony console doctrine:migrations:sync-metadata-storage
+	symfony console doctrine:migration:migrate --no-interaction
+
+## Database - Generate a migration from current db schema diff
+db.diff:
+	symfony console doctrine:migrations:diff
+
+## Database - Update (alias for migrate)
+db.update: db.migrate
+
+## Database - Force update the database schema
+db.update-force: db.create
```

Comment:
what about `make db.update FORCE=1` ? this is something we use deci-dela

Reply:
It seems a bit hard to discover and the code in the target would be a bit more complex. But no strong opinion on this.

Reply:
What would you think of this ? 

https://github.com/manala/manala-recipes/pull/291/commits/37d1a38e156efff31dfb4c60774469efdbcec828

Reply:
no need for shell syntax, `make` has its own one
```
$(if $(FORCE), blablabla, blibliblib)
```
but i just realize that `update-force` is not a forced `update` , but a `migrate`, and that is not the same limonade.

Reply:
> Having both update FORCE=1 and update-force does not really make sense :)

That's mainly to split up the related code, we can hide the `update-force` target.

> but i just realize that update-force is not a forced update , but a migrate, and that is not the same limonade.

`db.update-force` is a schema update --force. `db.migrate` is the one running migrations.